bcgov/cas-registration
# 661 parent operator addresses

card: https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=51034863

This card involved creating a `handle_operator_addresses` helper function that's used for both operators and parent operators. Sepehr and I had trouble with circular references in the tests, which is why so many of the import statements changed.

For manual testing:
part 1:
- create a new operator and add both business and mailing addresses, and add a parent operator with business and mailing addresses. Check db--should have four address records
- edit all addresses of this operator. In the db, we should have the 4 edited records, not any extra ones

part 2: 
- create a new operator with only a business address. There should be one address in the `Address` table and the operator physical and mailing address ids should be the same
- edit the operator and add a mailing address. now there should be two address, and the operator address ids should be different
